icuSources/common/udata.cpp

```diff
@@ -703,13 +703,15 @@ openCommonData(const char *path,          /*  Path from OpenChoice?          */
             }
 #if !defined(ICU_DATA_DIR_WINDOWS)
 // When using the Windows system data, we expect only a single data file.
+#if defined(USE_PACKAGE_DATA)
```

Comment:
Maybe just do `#if !defined(ICU_DATA_DIR_WINDOWS) && defined(USE_PACKAGE_DATA)` above and for the next change below instead? I see no reason to nest these #ifdefs when you are excluding the exact same code blocks.

Reply:
Done!